### PR TITLE
Fix changed google cloud URLs

### PR DESCRIPTION
**Pull request by nickandrew**
All documentation has been moved to cloud.google.com.

Signed-off-by: Nick Andrew nick@nick-andrew.net

### CODE REVIEW

**Comment by thaJeztah** [comment]
replaces https://github.com/docker/docker/pull/13434, which already got 1 "LGTM"

File: docs/installation/google.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ parent = "smn_cloud"
 Read more about [deploying Containers on Google Cloud Platform][5].
 
 [1]: https://cloud.google.com/console
-[2]: https://developers.google.com/compute/docs/signup
-[3]: https://developers.google.com/cloud/sdk
-[4]: https://developers.google.com/compute/docs/containers#container-optimized_google_compute_engine_images
-[5]: https://developers.google.com/compute/docs/containers
+[2]: https://cloud.google.com/compute/docs/signup
```
Comment (thaJeztah) [inline docs/installation/google.md]: This redirected to https://cloud.google.com/compute/docs/quickstart for me, or is that if you're logged in already?

File: docs/installation/google.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ parent = "smn_cloud"
...
+[3]: https://cloud.google.com/sdk
```
Comment (thaJeztah) [inline docs/installation/google.md]: nit: it looks like Google wants a trailing slash here in this URL, https://cloud.google.com/sdk/

File: docs/installation/google.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ parent = "smn_cloud"
 Read more about [deploying Containers on Google Cloud Platform][5].
 
 [1]: https://cloud.google.com/console
```
Comment (thaJeztah) [inline docs/installation/google.md]: This redirects me to https://console.developers.google.com/project  not sure what the canonical URL is?

**Comment by thaJeztah** [comment]
@nickandrew thanks! I noticed a couple of redirects when I tried those URLs. Could you verify if they redirect for you as well, and update them (if nescessary)?

**Comment by thaJeztah** [comment]
ping @nickandrew could you address my comments?

**Comment by calavera** [comment]
It looks like the old urls redirect to the right places for us. I'm going to close this PR. @nickandrew feel free to reopening it when the comments are adressed.

**Pull request closed without merging**